Admin profile page

MarcoSaba71ni opened the pull request:
### What does this PR do?

### Description of Task to be completed?

### How should this be manually tested?

### Any background context you want to provide?

MarcoSaba71ni committed:
Your commit message

MarcoSaba71ni committed:
Final Page

MarcoSaba71ni committed:
draft-admin-profile page

In order not to create a disorder inside the folders, the file that would better match the Figma design would be draft-admin-profile-page.

MarcoSaba71ni committed:
new admin-profile page

GustavBoe reviewed (changes requested):
Looking really good Marco! Here are your points:

### For desktop

**Other**
- [x] Change the title of the page to mirror the file name

**Breadcrumbs**
- [x] Ensure that the gap between header and breadcrumbs matches Figma.
- [x] Check that the gap between the breadcrumbs are equal to 1.5rem.
- [x] Change the text from "Dashboad" to "Dashboard".
- [x] Give the text a tracking-[0.015rem]

**Container**
- [ ] Change the top margin from mt-40 to mt-[5rem].
- [ ] Change the width of the border to 3px.
- [ ] Ensure that the gap between text and button is around 2.5rem.
- [ ] Increase the gap between the button and  container-border to be equivalent to 9.25rem (if this looks off, just try to mirror Figma).


### For mobile

**Breadcrumbs**
- [ ] Give the breadcrumb a top margin of 1.19rem

**Container**
- [ ] Decrease the top margin to be equivalent to 4.84rem, moving it closer to the breadcrumbs.

**Footer**
- [x] Make sure that you have the correct path to the image. Looks like it misses some "_"´s.

MarcoSaba71ni commented:
Hey, Gustav. I am having some issues with those cases:

 Change the top margin from mt-40 to mt-[5rem].
   When I write -5[rem] it goes all the way up having no padding at all.
 Change the width of the border to 3px.
   When I write border-3, the border simply disappears.

MarcoSaba71ni committed:
Final Changes

MarcoSaba71ni commented:
- Changed the gaps between the breadcrumb and the header using top padding of 80px
- Spaced the breadcrumbs (including the arrow) using padding x of 24 px
- Changed it to "Dashboard"
- Switching the margin top from 160px to 5rem, created an error where ended up with no margin. For that reason I've set the my-20 (as the margin on the bottom would be the same value). Still it is not working. 
- When changing the width of the board to 3px, border-[3px] the border wouldn't be visible.

Anything related to the mobile container board is not working well- mainly because my tailwind becomes unresponsive when I apply spacing using -[px]

In order to meet the padding and margin requirements when it comes to the breadcrumbs, I decided to create two different divs: one for large and medium screens and the other for small screens.

Not being able to adjust the padding between the breadcrumb and the board (move it closer to the breadcrumb)

MarcoSaba71ni committed:
Unfinished changes

Please, read the last comments

GustavBoe committed:
Merge branch 'develop' into admin-profile-page

GustavBoe reviewed (approved):
I understand your challanges Marco! I will approve so that Fabian can start on his tasks.

GustavBoe committed:
Merge branch 'develop' into admin-profile-page
